### PR TITLE
Moving more schematics

### PR DESCRIPTION
**Pull request by aallan**
😢

### CODE REVIEW

File: hardware/raspberrypi/schematics/README.md [hunk 1]
```diff
@@ -4,39 +4,39 @@ Schematics for the various Raspberry Pi board versions:
...
 
 ## Raspberry Pi 3 Model B
 
-- [Revision 1.2](rpi_SCH_3b_1p2_reduced.pdf)
+- [Revision 1.2](https://datasheets.raspberrypi.org/rpi3/raspberry-pi-3-b-reduced-schematics.pdf)
 
 ## Raspberry Pi 2 Model B
 
-- [Revision 1.2](rpi_SCH_2b_1p2_reduced.pdf)
+- [Revision 1.2](https://datasheets.raspberrypi.org/rpi/raspberry-pi-2-b-reduced-schematics.pdf)
```
Comment (lurch) [inline hardware/raspberrypi/schematics/README.md:19]: Why isn't this `/rpi2/` ?

**Comment by JamesH65** [comment]
Won't all this sort of thing be moving in to PIP?

**Comment by aallan** [comment]
> Won't all this sort of thing be moving in to PIP?

Eventually yes. But that'll be separate movement of stuff and things later. For now I'm staging them from datasheets.raspberrypi.org. Eventually we move them into PIP as source-of-truth. But not right now.

File: hardware/raspberrypi/schematics/README.md (hunk 1 above)
Comment (aallan) [reply hardware/raspberrypi/schematics/README.md:19]: I guess it should be.

Comment (aallan) [reply hardware/raspberrypi/schematics/README.md:19]: Fixed on master, a307399.

**Pull request merged**